packages/storage/src/database/common/merkle-tree.ts

```diff
@@ -18,12 +18,20 @@ export type TMerkleProof = {
   isLeft: boolean;
 };
 
-export class ModelMerkleTree extends ModelGeneral<MerkleProof> {
+export type TMerkleNode = {
+  nodeId: ObjectId;
+  timestamp: Date;
+  hash: string;
+  level: number;
+  index: number;
+};
+
+export class ModelMerkleTree extends ModelGeneral<TMerkleNode> {
   private static instances = new Map<string, ModelMerkleTree>();
 
   private zeroes!: Field[];
 
-  private height!: number;
+  private _height!: number;
```

Comment:
I think default must be 0, dont using null assertion in this case.
Typescript yield you because you don't have set height in constructor. So you have default values init for this height

Reply:
Done